Update KILL TIDB descriptions for v5.3 (#15612)
pingcap/docs: sql-statements/sql-statement-kill.md

+------+------+-----------+------+---------+------+-------+------------------+
2 rows in set (0.00 sec)

> You can get the TiDB Server details of a killed session by querying the `INSTANCE` column of `INFORMATION_SCHEMA.CLUSTER_PROCESSLIST`:

Reviewer: ```suggestion
You can get the TiDB Server details of a session to be killed by querying the `INSTANCE` column of `INFORMATION_SCHEMA.CLUSTER_PROCESSLIST`:
```

```sql
SELECT ID, USER, INSTANCE, INFO FROM INFORMATION_SCHEMA.CLUSTER_PROCESSLIST;
```

```sql
+---------------------+------+-----------------+-----------------------------------------------------------------------------+
| ID | USER | INSTANCE | INFO |
+---------------------+------+-----------------+-----------------------------------------------------------------------------+
| 1 | root | 127.0.0.1:10082 | SELECT ID, USER, INSTANCE, INFO FROM INFORMATION_SCHEMA.CLUSTER_PROCESSLIST |
| 2 | root | 127.0.0.1:10080 |  |
+---------------------+------+-----------------+-------------------------------------------------------------
2 rows in set (0.00 sec)
```

> To execute the `KILL TIDB` statement, connect to the same TiDB node as the killed session:

Reviewer: ```suggestion
To execute the `KILL TIDB` statement, connect to the TiDB node where the session needs to be killed:
```
